platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -807,6 +807,6 @@ public void setPlatformState(@NonNull final PlatformState platformState) {
     @NonNull
     @Override
     public String getInfoString(final int hashDepth) {
-        return State.createInfoString(hashDepth, getPlatformState(), getHash(), this);
+        return com.swirlds.platform.state.State.createInfoString(hashDepth, getPlatformState(), getHash(), this);
```

Comment:
Unfortunate that we have multiple classes named `State`...

Reply:
It is but the other `State` will be deprecated for production use soon.